joschahenningsen/TUM-Live
# Make theater mode default

**Description:**

To avoid unused space within the watch page, this PR sets the theater mode as a default for the player.

`{if $course.ChatEnabled}}md:w-4/6{{end}}` was removed since it is overwritten by the current theater mode implementation anyways and didn't have any noticeable effect.

**Steps for Testing:**

- Check that untoggling still works
- Check that it behaves correctly on different aspect ratios
- Check that it works on streams with a Chat
